README.md

```diff
@@ -90,7 +90,7 @@ Documentation, included in this repo in the root directory, is built with [Jekyl
 ### Running sample site locally
 
 1. Install Ruby and run `gem install bundler`.
-1. Install node.js.
+1. Install node.js - version 22 required.
```

Comment:
Also meant in the release notes - it's not just required for running the sample site, but for building a custom bundle too (which many of our apps do)